src/importer/onnx/ops/pool.cpp

```diff
@@ -52,15 +52,25 @@ namespace
 
 void onnx_importer::convert_op_AveragePool(const NodeProto& node)
 {
-    convert_pool(node, reduce_mean, 0.f);
+    convert_pool<>(node, reduce_mean, 0.f);
+}
+
+void onnx_importer::convert_op_GlobalAveragePool(const NodeProto& node)
+{
+    convert_pool<true>(node, reduce_mean, 0.f);
 }
 
 void onnx_importer::convert_op_MaxPool(const NodeProto& node)
 {
-    convert_pool(node, reduce_max, numeric_limits<float>::max());
+    convert_pool<>(node, reduce_max, 0.f);
```

Comment:
Is it right to set init_value = 0?

Reply:
Hmm.. you're probably right, should be the lowest value, otherwise works as expected only for non-negative values. Will fix it now.

Reply:
Fixed.